wpilibc/src/main/native/include/frc/InterruptableSensorBase.h

```diff
@@ -144,8 +144,8 @@ class InterruptableSensorBase : public ErrorBase, public SendableBase {
   virtual void SetUpSourceEdge(bool risingEdge, bool fallingEdge);
 
  protected:
-  // atomic for proper destruction
-  std::atomic<HAL_InterruptHandle> m_interrupt{HAL_kInvalidHandle};
+  HAL_InterruptHandle m_interrupt{HAL_kInvalidHandle};
+  std::unique_ptr<InterruptEventHandler> m_interruptHandler{nullptr};
```

Comment:
It no longer needs to be atomic?

Reply:
The atomic was added when I was worried about destruction races, however I realized the atomic was being used incorrectly anyway for that purpose and was basically useless.